apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-responsive-padding-example.component.html

```diff
@@ -0,0 +1,133 @@
+            delectus necessitatibus incidunt numquam asperiores tenetur iure. Cum consequuntur impedit repellendus esse,
+            facere autem optio consequatur nobis?</span
+        >
+    </fdp-dynamic-page-header>
+    <fdp-dynamic-page-content id="basic">
+        <!-- page content goes here -->
+        Normal Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit sunt laboriosam totam maiores.
+        Unde quas quis soluta adipisci mollitia distinctio. Natus nam repellat, aut ad et tempore cum. Corporis, nobis.
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam ad provident similique, voluptatibus
+        obcaecati perferendis excepturi.
+        <br />
+        <br />
+        <br />
+        Velit officiis sunt, debitis eum perspiciatis consectetur? Consequatur consequuntur quia nisi iste tempora
+        aspernatur. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque architecto dignissimos reprehenderit
+        perspiciatis excepturi voluptate. Vel enim eius doloribus amet vitae debitis. Similique vel ipsam debitis fuga
+        sequi eveniet perspiciatis?
+        <br />
+        <br />
+        <br />
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quaerat veniam libero nostrum explicabo sed odit?
+        Recusandae praesentium voluptatum cum omnis, placeat beatae quasi eum odio doloribus inventore, eos iusto sed.
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit sunt laboriosam totam maiores. Unde quas
+        quis soluta adipisci mollitia distinctio. Natus nam repellat, aut ad et tempore cum. Corporis, nobis. Lorem
+        ipsum dolor sit amet consectetur adipisicing elit. Laboriosam ad provident similique, voluptatibus obcaecati
+        perferendis excepturi.
+        <br />
+        <br />
+        <br />
+        Velit officiis sunt, debitis eum perspiciatis consectetur? Consequatur consequuntur quia nisi iste tempora
+        aspernatur. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque architecto dignissimos reprehenderit
+        perspiciatis excepturi voluptate. Vel enim eius doloribus amet vitae debitis. Similique vel ipsam debitis fuga
+        sequi eveniet perspiciatis?
+        <br />
+        <br />
+        <br />
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quaerat veniam libero nostrum explicabo sed odit?
+        Recusandae praesentium voluptatum cum omnis, placeat beatae quasi eum odio doloribus inventore, eos iusto sed.
+        <br />
+        <br />
+        <br />
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Minus eligendi dolore omnis veritatis! Et voluptatibus
+        error, commodi, dignissimos nostrum nemo deleniti est cumque fuga reiciendis, fugiat impedit explicabo
+        voluptates at? Normal
+        <br />
+        <br />
+        <br />
```

Comment:
these were intentionally added because Floating Footer Bar styles went through some changes where it is now `position:absolute` in Styles making the Bar stick to the bottom of the page. A fix for it right now is to add `position:relative` but it goes on top of the content. Will additionally apply `margin-top: 3rem` as in this PR https://github.com/SAP/fundamental-ngx/pull/3469 to avoid using `<br>`'s